CacheManager.StackExchange.Redis.SelfCleaning.Tests/SelfCleaningRedisIntegrationTests.cs

```diff
@@ -0,0 +1,247 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using CacheManager.Core;
+using CacheManager.Core.Internal;
+using CacheManager.StackExchange.Redis.SelfCleaning.Core;
+using CacheManager.StackExchange.Redis.SelfCleaning.Timers;
+using Moq;
+using NUnit.Framework;
+using StackExchange.Redis;
+
+namespace CacheManager.StackExchange.Redis.SelfCleaning.Tests
+{
+    [TestFixture]
+    public class SelfCleaningRedisIntegrationTests
+    {
+        private const double TIME_TO_LIVE_MILLISECONDS_DIFFERENCE_THRESHOLD = 100;
```

Comment:
Too small values might cause failure of tests in some computers.. I suggest you increase this to 500 ms (it worked on my computer for 5 consecutive runs). Or even 1 second to be sure? Choose 😄

Reply:
500ms feels like too much. I think beyond a certain value it loses the concept of realtime. For now I've set it to 200ms, but try checking what is the lowest value for which the test consistently pass on your PC.

Reply:
150ms passes 7 times in a row

Reply:
I think we can keep it at 200 then